web-ide/proxy/static/js/main.js

```diff
@@ -16,7 +16,7 @@ function acceptClick(event) {
 function setCookie(sKey, sValue, sPath, sDomain, protocol) {
     if (!sKey || /^(?:expires|max\-age|path|domain|secure)$/i.test(sKey)) { return false; }
     var sExpires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
-    var bSecure = protocol.startsWith("https:") ? true : false
+    var bSecure = protocol.substring(0,6) === "https:";
```

Comment:
Shouldn’t `protocol` be equal to `https:` instead of only starting with `https:`? Then we can just replace this by `protocol === "https:"`.

Reply:
👍